src/Files.App/Data/Models/GitItemModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using LibGit2Sharp;
+
+namespace Files.App.Data.Models
+{
+	/// <summary>
+	/// Represents an model for Git items
+	/// </summary>
+	internal class GitItemModel
+	{
+		/// <summary>
+		/// Gets or initializes file change kind
+		/// </summary>
+		/// <remarks>
+		/// This is often showed as A(Add), D(Delete), M(Modified), U(Untracked) in VS Code.
+		/// </remarks>
+		public ChangeKind Status { get; init; }
+
+		/// <summary>
+		/// Gets or initializes file change kind humanized string
+		/// </summary>
+		/// <remarks>
+		/// This is often showed as A(Add), D(Delete), M(Modified), U(Untracked) in VS Code.
+		/// </remarks>
+		public string? StatusHumanized { get; init; }
```

Comment:
Instead of having an extra value in the model, maybe we can use a converter to display `Status` as a string.

Reply:
Is that really extra info? If you think so, we should remove DateTimeOffset properties first. I strongly think that I have to keep original data even on the item model that is bound to the XAML.

Reply:
It is not extra info. You have also DateTimeOffset properties in ListedItem, which are not used in UI.

Reply:
Status is ChangeKind enum such as 'Ignored', 'Modified', 'Added'.